bin/crm

```diff
@@ -37,8 +37,9 @@ except ImportError as msg:
     sys.stderr.write('''Fatal error:
     %s
 
-Failed to start crmsh! This is likely due to a broken
-installation or a missing dependency.
+Failed to start crmsh! This is likely due to:
+- A missing dependency (corresponding python3 version)
```

Comment:
```suggestion
- A missing dependency (eg. corresponding python3 version)
```

Reply:
For the commit title, a little bit more specific, how about,

s/Dev: crm: Improve error hints/Dev: crm: Improve error hints when starting crmsh/

Reply:
Changed
Thanks!